service/src/main/java/com/ge/predix/acs/attribute/cache/AbstractAttributeCache.java

```diff
@@ -0,0 +1,62 @@
+package com.ge.predix.acs.attribute.cache;
+
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.ge.predix.acs.model.Attribute;
+import com.ge.predix.acs.policy.evaluation.cache.AbstractPolicyEvaluationCache;
+
+@Component
+public abstract class AbstractAttributeCache implements AttributeCache {
```

Comment:
Done. We can make the relevant config changes to bump up the app memory in acs-ci-config as part of US87168. Also, as discussed, Redis-related changes will be pulled out to the US87168 branch.